TomTomSDKExamples/Examples/UseCase/Offline/OfflineNavigationContent.swift

```diff
@@ -0,0 +1,689 @@
+    func didDeviateFromRoute(currentRoute _: TomTomSDKRoute.Route, location _: TomTomSDKLocationProvider.GeoLocation) {}
+
+    func didProposeRoutePlan(routePlan _: TomTomSDKNavigationEngines.RoutePlan, reason _: TomTomSDKNavigationEngines.RouteReplanningReason) {}
+
+    func didReplanRoute(replannedRoute: TomTomSDKRoute.Route, reason _: TomTomSDKNavigationEngines.RouteReplanningReason) {
+        displayedRouteSubject.send(replannedRoute)
+    }
+
+    func didChangeRoutes(navigatedRoutes _: TomTomSDKNavigation.NavigatedRoutes) {}
+
+    func didReplanRouteOnLanguageChange(replannedRoute _: TomTomSDKRoute.Route, reason _: TomTomSDKNavigationEngines.RouteReplanningReason, language _: Locale) {}
+}
+
+// MARK: - MainView
+
+struct OfflineMainView: View {
+    // MARK: Internal
+
+    var body: some View {
+        ZStack(alignment: .bottom) {
+            TomTomOfflineMapView(contentInsets: $contentInsets, offlineNavigationController: offlineNavigationController)
+            if offlineNavigationController.showNavigationView {
+                NavigationView(
+                    offlineNavigationController.navigationViewModel,
+                    action: offlineNavigationController.onNavigationViewAction
+                )
+            }
+        }
+        .onDisappear {
+            offlineNavigationController.stopNavigating()
+        }
+    }
+
+    // MARK: Private
+
+    @State
+    private var contentInsets: EdgeInsets = .init()
+
+    @StateObject
+    var offlineNavigationController = OfflineNavigationController()
+}
+
+// MARK: - TomTomOfflineMapView
+
+struct TomTomOfflineMapView {
+    var mapView: MapView!
+    var contentInsets: Binding<EdgeInsets>
+    var offlineNavigationController: OfflineNavigationController
+
+    func updateEdgeInsets(context: Context) {
+        context.coordinator.setContentInsets(edgeInsets: contentInsets.wrappedValue)
+    }
+
+    init(contentInsets: Binding<EdgeInsets>, offlineNavigationController: OfflineNavigationController) {
+        self.contentInsets = contentInsets
+        self.offlineNavigationController = offlineNavigationController
+
+        /// These paths are required for the offline map configuration
+        guard let updateStoragePath = OfflineMapPath.updateStoragePath,
+              let persistantStoragePath = OfflineMapPath.persistantStoragePath,
+              let updateServerURL = URL(string: OfflineConfig.ndsUpdateServer),
+              let mapDataPath = OfflineMapPath.mapDataPath,
+              let keystorePath = OfflineMapPath.keystorePath
+        else {
+            /* YOUR CODE GOES HERE */
+            fatalError("Error!")
+        }
+
+        /// In the following code, an "NDSStoreUpdateConfig" object will be created that includes configuration, such as relevant region updates.
+        /// Additionally, the empty map folders were copied by calling "OfflineMapHelper.copyFolders()" in the "OfflineNavigationView.swift" file.
```

Comment:
nit: a standard way to mark code or code symbols is by using backticks.

```suggestion
        /// Additionally, the empty map folders were copied by calling `OfflineMapHelper.copyFolders()` in the `OfflineNavigationView.swift` file.
```